src/invocation/InvocationService.ts

```diff
@@ -13,6 +14,7 @@ class InvocationService {
 
     private correlationCounter = 0;
     private pending: {[id: number]: Q.Deferred<ClientMessage>} = {};
+    private eventHandlers: {[id: number]: (...args: any[]) => any} = {};
```

Comment:
I thought there is no point in moving a single event handler list to another service. This way nodejs resembles other clients' architecture.